STYLEGUIDE.adoc

```diff
@@ -0,0 +1,578 @@
+= Plutus Haskell style guide
```

Comment:
"Plutus Apps" or "Plutus Tools"?

Reply:
Yes it was shamelessly copied/pasted from plutus-core, I guess "repo-name Style guide" should be fine?